src/ctapipe_io_lst/calibration.py

```diff
@@ -356,35 +364,77 @@ def _read_calibration_file(path):
         Read the correction from hdf5 calibration file
         """
 
-        with tables.open_file(path) as f:
-            tel_groups = [
-                key for key in f.root._v_children.keys()
-                if key.startswith('tel_')
-            ]
-
         mon = MonitoringContainer()
 
-        for base in tel_groups:
-            with HDF5TableReader(path) as h5_table:
-                # read the calibration data
-                tel_id = int(base[4:])
-                mon.tel[tel_id] = MonitoringCameraContainer(
-                    calibration=next(h5_table.read(f'/{base}/calibration', WaveformCalibrationContainer)),
-                    pedestal=next(h5_table.read(f'/{base}/pedestal', PedestalContainer)),
-                    flatfield=next(h5_table.read(f'/{base}/flatfield', FlatFieldContainer)),
-                    pixel_status=next(h5_table.read(f"/{base}/pixel_status", PixelStatusContainer)),
-                )
+        if path.name.endswith(".h5"):
+            # get keys in file
+            with tables.open_file(path) as f:
+                tel_groups = [
+                    key for key in f.root._v_children.keys()
+                    if key.startswith('tel_')
+                ] 
+
+            for base in tel_groups:
+                with HDF5TableReader(path) as h5_table:
+                    # read the calibration data
+                    tel_id = int(base[4:])
+                    mon.tel[tel_id] = MonitoringCameraContainer(
+                        calibration=next(h5_table.read(f'/{base}/calibration', WaveformCalibrationContainer)),
+                        pedestal=next(h5_table.read(f'/{base}/pedestal', PedestalContainer)),
+                        flatfield=next(h5_table.read(f'/{base}/flatfield', FlatFieldContainer)),
+                        pixel_status=next(h5_table.read(f"/{base}/pixel_status", PixelStatusContainer)),
+                    )
+        
+        elif path.name.endswith(".fits") or path.name.endswith(".fits.gz"):
+            
+            CALIB_CONTAINERS = {
+                "calibration": WaveformCalibrationContainer,
+                "flatfield": FlatFieldContainer,
+                "pedestal": PedestalContainer,
+                "pixel_status": PixelStatusContainer,
+            } 
+
+            mon_data = MonitoringCameraContainer()
+         
+            with fits.open(path) as f:
+                tel_id = f['PRIMARY'].header['TEL_ID']
+                for key in CALIB_CONTAINERS.keys():
+                    table = QTable.read(f, hdu=key)
```

Comment:
I thought you store image hdus now?

Reply:
Non, these are BinTableHDU

Reply:
Why are we storing calibration in bintablehdu and everything else in image hdus?

Reply:
I think this should be consistent, so that EVB and we can implement a single interface just using different extension and data names.

Reply:
Now it is like that and Dirk did use dit already like that